segment-appcues/segment-appcues.properties

```diff
@@ -1,6 +1,9 @@
 GROUP_ID = com.appcues
 ARTIFACT_ID = segment-appcues
-VERSION = 1.0.0-beta05
+VERSION_MAJOR = 1
+VERSION_MINOR = 0
+VERSION_PATCH = 0
+VERSION_CLASSIFIER = beta04
```

Comment:
this is intentionally moved back to `beta04` (current value) so that the next release can bump it up to 05 as part of the script.